Nautono/RTOS_framework_v1/src/sail_eeprom.c

```diff
@@ -525,4 +532,176 @@ static uint16_t Idx2Addr(uint16_t idx)
 	return EEPROM_START_OFFSET + EEPROM_ENTRY_OFFSET + EEPROM_ENTRY_LENGTH *idx;
 }
 
+#define MISSION_ENTRY_COUNT		40
```

Comment:
What is your view on adding ```#define``` macros in the middle of the source file. I'm not sure this is consistent with the other source files. 

All I'm saying is that we should be consistent with the style and where things are defined.